cloudflare.services.yml

```diff
@@ -11,3 +11,8 @@ services:
     arguments: ['%cloudflare.cache_tag_header_limit%']
     tags:
       - { name: event_subscriber }
+  cloudflare:
```

Comment:
i'd rename this cloudflare.service

One think you can think of is "what" the service is doing. Maybe its a manager or maybe its an engine or something. But, you may have a better term for this than service which is generic